hadoop-pipeline-application/src/main/java/org/molgenis/hadoop/pipeline/application/cachedigestion/SamplesInfoFileField.java

```diff
@@ -52,15 +55,15 @@ static SamplesInfoFileField getEnum(String fieldString)
 	}
 
 	/**
-	 * Compares the name of this {@link Enum} (excluding the {@link Enum} type name) to a {@link String}. Ignores
+	 * Compares the name of this {@link Enum} (excluding the {@link Enum} class name) to a {@link String}. Ignores
 	 * capitalization during the comparison!
 	 * 
 	 * @param fieldName
 	 *            {@link String}
-	 * @return {@code true} if the {@link String} equals the name of the {@link Enum} (ignoring the {@link Enum} type
+	 * @return {@code true} if the {@link String} equals the name of the {@link Enum} (ignoring the {@link Enum} class
 	 *         name), otherwise false.
 	 */
-	boolean nameEquals(String fieldName)
+	public boolean nameEquals(String fieldName)
 	{
 		return this.toString().equals(fieldName.toUpperCase());
```

Comment:
Original implementation was because only some enum instances have a differing name compared to the enum instance name itself. However, this solution is indeed much more readable and will be implemented.